devel/grpc/Portfile

```diff
@@ -1,53 +1,76 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem                                     1.0
-PortGroup       github                         1.0
-
-github.setup    grpc grpc 1.22.0 v
-revision        2
-categories      devel
-maintainers     nomaintainer
-license         Apache-2
-description     A high performance, open-source universal RPC framework.
-homepage        https://grpc.io
-platforms       darwin
+PortSystem          1.0
+PortGroup           github 1.0
 
-# ignore pre releases
-github.livecheck.regex {([0-9.]+)}
+github.setup        grpc grpc 1.30.2 v
+revision            0
+categories          devel
+maintainers         nomaintainer
+license             Apache-2
+fetch.type          git
 
```

Comment:
You don't usually need this if you use the github PG -- is there something weird about this port to need it?